pom.xml

```
@@ -42,59 +42,50 @@
   <properties>
     <revision>0.6.3</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.145</jenkins.version> <!-- to pick up https://github.com/jenkinsci/jenkins/pull/3662 -->
+    <jenkins.version>2.164.3</jenkins.version> <!-- to pick up https://github.com/jenkinsci/jenkins/pull/3662 -->
```

nit: The comment is unnecessary now that we are on an LTS baseline.

```suggestion
    <jenkins.version>2.164.3</jenkins.version>
```